backend/src/Notifo.Domain/Integrations/IntegrationExtensions.cs

```diff
@@ -6,13 +6,25 @@
 // ==========================================================================
 
 using Notifo.Domain.Users;
+using Notifo.Infrastructure.Collections;
 
 namespace Notifo.Domain.Integrations;
 
 public static class IntegrationExtensions
 {
     public static UserInfo ToContext(this User user)
     {
-        return new UserInfo { Id = user.Id, EmailAddress = user.EmailAddress, PhoneNumber = user.PhoneNumber, Properties = user.SystemProperties };
+        return new UserInfo
+        {
+            Id = user.Id,
+            EmailAddress = user.EmailAddress,
+            PhoneNumber = user.PhoneNumber,
+            Properties = user.Properties
```

Comment:
I am not a big fan of LINQ in this case. Can we just make it with a normal dictionary like

var properties = new Dictionary<string, string>(user.Properties);

if (user.SystemProperties != null)
{
   foreach (var (key, value) in user.SystemProperties)
   {
      properties[key] = values;
   } 
}

Reply:
Sure, of course. Pushing the new changes...